frontend/app.py

```diff
@@ -1,22 +1,32 @@
 import os
-
-
-from dotenv import load_dotenv, find_dotenv
 import requests
 import streamlit as st
+from dotenv import load_dotenv, find_dotenv
+
+from pages.project_issues_chat import project_issues_chat
+from pages.issue_summary import issue_summary
```

Comment:
These imports are unused. Streamlit automatically addes pages to sidebar and may also trigger an exception if code in the other pages runs before `set_page_config` `set_page_config() can only be called once per app , and must be called as the first Streamlit command in your script` 
```suggestion
```